setup.py

```diff
@@ -88,6 +88,7 @@ def package_files(prefix: str, directory: str, extension: str):
     'packaging>=21.3.0,<23',
     'importlib-metadata>=5.0.0,<7',
     'mosaicml-cli>=0.4.12,<0.5',
+    'py3nvml>=0.2.7'
```

Comment:
Should we move this to a separate target for this callback? I've seen some weird issues w nvml and docker images in past... but not 100% sure on this. @hanlint  do you remember what you were seeing?

Reply:
Good point. I made it a separate target and used the same python nvml lib that `health_check.py` is using.